.dockerignore

```diff
@@ -31,3 +31,172 @@ dask*-space/
 
 # mypy cache
 **/.mypy_cache/
+
+

```

Comment:
just a thought but: the idea with dockerignore files I think are:
- ignore everything ```*```
- then only add what is necessary as this is used in the docker build to set up the context and might make the build magnitudes faster. I guess I will review this one of these Fridays

Reply:
> I've got a few questions:
> 
> 1. We already have [ci-service-integration-library](https://github.com/ITISFoundation/ci-service-integration-library) which is published to GitHub that already contains `ooil`.
> 2. Why is this an image that is not part of simcore released with simcore?
> 3. Where it will be pushed, how it will be versioned?
> 4. How would this be used?

discussed offline

Reply:
> just a thought but: the idea with dockerignore files I think are:
> 
> * ignore everything `*`
> * then only add what is necessary as this is used in the docker build to set up the context and might make the build magnitudes faster. I guess I will review this one of these Fridays

In principle I agree with this approach but IMO there is a flaw. Say that you want to include ``packages/servicelib``. Using your approach you exclude everything except whatever is in that folder. The problem is, what happens e.g. with all the artifacts produced for python development (i.e. __cache__, eggs, etc) inside of that folder? Shouldn't it make sense that the ``.dockerignore`` includes as well the ``.gitignore``?